lib/request-handler.js

```diff
@@ -13,6 +13,24 @@ const getCrossOriginHeader = (fragmentUrl, host) => {
     return '';
 };
 
+// Early preloading of primary fragments assets to improve Performance
+const getAssetsToPreload = ({ link }, { headers = {} }) => {
+    let assetsToPreload = '';
+    const links = parseLinkHeader(link);
+    // Handle Server rendered fragments without depending on assets
+    if (links == null) {
+        return assetsToPreload;
+    }
+    if (links.stylesheet && links.stylesheet.url) {
+        assetsToPreload += `<${links.stylesheet.url}>; rel="preload"; as="style"; nopush,`;
```

Comment:
Yes and browsers does ignore it.. Don't think we need to do anything here. Thoughts?